experimenter/tests/integration/nimbus/test_remote_settings.py

```diff
@@ -254,3 +255,29 @@ def test_rollout_live_update_reject_on_experimenter(
     summary_page.set_rejection_reason()
     summary_page.submit_rejection()
     summary_page.wait_for_rejection_notice_visible()
+
+
+@pytest.mark.remote_settings
+@pytest.mark.skipif(
+    os.getenv("PYTEST_ARGS") != "FIREFOX_DESKTOP",
```

Comment:
Actually now that I think about it we probably will wanna make this an inclusive check for the mobile apps only since we'll start adding webapps and firstrun doesn't apply there so then you'd wanna say something like

```py
os.getenv("PYTEST_ARGS") not in [.....]
```